### PR TITLE
Homework1: Yauheni Kachkayeu

### PR DESCRIPTION
**Pull request by KACHKAYEUYAUHENI**
### PR checklist
Please make sure you've passed PR checklist
- [ ] my PR passes CI checks (flake8 and pycodestyle).
- [ ] my PR named according to guidelines.
- [ ] my PR contains complete tasks descriptions for all tasks.
- [ ] my PR made from the fork feature branch to appropriate branch.
- [ ] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

**Comment by KACHKAYEUYAUHENI** [comment]
homework1

File: src/task9.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from time import localtime
+
```
Comment (belovmd) [inline src/task9.py:3]: 1 line between import and code

File: src/task8.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# indent your Python code to put into an email
+import glob
```
Comment (belovmd) [inline src/task8.py:3]: 0 lines between import and code

File: src/task13.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import itertools
+
+
```
Comment (belovmd) [inline src/task13.py:4]: 2 lines between import and code

File: src/task10.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+REFRAIN = '''
+%d bottles of beer on the wall,
+%d bottles of beer,
+take one down, pass it around,
+%d bottles of beer on the wall!
+'''
+bottles_of_beer = 9
```
Comment (belovmd) [inline src/task10.py:9]: should be at least 1 empty line between code and constant

**Review by belovmd — CHANGES_REQUESTED** [review]
failed sanity checks - wrong PR name
failed sanity checks - missed docstrings

**Review by belovmd — APPROVED** [review]
1

**Pull request closed without merging**